src/libraries/Microsoft.PowerFx.Interpreter/Functions/LibraryText.cs

```diff
@@ -253,7 +251,7 @@ public static bool TryText(FormattingInfo formatInfo, IRContext irContext, Formu
                 return false;
             }
 
-            Contract.Assert(AllowedListConvertToString.Contains(value.Type));
+            Contract.Assert(StringValue.AllowedListConvertToString.Contains(value.Type));
```

Comment:
>StringValue.AllowedListConvertToString.Contains(value.Type) [](http://example.com/codeflow?start=28&length=59)

nit: What you think this instead?
value.CanCoerceToStringValue()

Reply:
dont want to go around since typecoerciosnprovider use library text, now libary text back to use function in typecoercionprovider.